Jenkinsfile

```diff
@@ -36,10 +37,10 @@ pipeline {
                     // granted the access to the signing certificates via
                     // the machine's UI (remotely or directly), then
                     // the uploading to AppStoreConnect started to work.
-                    sh 'bin/archive.sh "Multisig - Staging Rinkeby"'
-                    sh 'bin/archive.sh "Multisig - Staging Mainnet"'
-                    sh 'bin/archive.sh "Multisig - Production Rinkeby"'
-                    sh 'bin/archive.sh "Multisig - Production Mainnet"'
+                    sh 'INFURA_KEY="${INFURA_STAGING_KEY}" bin/archive.sh "Multisig - Staging Rinkeby"'
```

Comment:
since the config.sh is used in tests, too, please add the same change to the line 18/19

Reply:
done